ios/Packet Tunnel Provider/Packet Tunnel Provider.entitlements

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.developer.networking.networkextension</key>
+	<array>
+		<string>packet-tunnel-provider</string>
+	</array>
+	<key>com.apple.developer.networking.vpn.api</key>
+	<array>
+		<string>allow-vpn</string>
+	</array>
+	<key>com.apple.security.application-groups</key>
+	<array>
+		<string>group.com.niels-andriesse.loki-network</string>
```

Comment:
The group should use loki-controlled namespace.

Team Apple Developer accounts have also been discussed with Simon before, which he agreed to, if there are concerns around ease of signing and such.

Reply:
Of course, this is just a temporary fix to be able to use network extensions. We'll switch this over once the team developer account is available.